### PR TITLE
Remove code that changes the proxy env. variable in crc status

### PR DESCRIPTION
**Pull request by guillaumerose**
Small cleanup

### CODE REVIEW

File: cmd/crc/cmd/oc_env.go [hunk 1]
```diff
@@ -27,12 +27,6 @@ func runOcEnv(args []string) error {
 		return fmt.Errorf("Error running the oc-env command: %s", err.Error())
 	}
 
-	client := newMachine()
-	consoleResult, err := client.GetConsoleURL()
-	if err != nil {
-		return err
-	}
-	proxyConfig := consoleResult.ClusterConfig.ProxyConfig
```
Comment (cfergeau) [inline cmd/crc/cmd/oc_env.go:35]: Is `NoProxy` correct without this? `NoProxy` needs to contain `api.xxx`, in order to know what `xxx` is, we need the bundle which is being used, in order to know which bundle is currently being used, we need to look at the machine metadata.

Comment (guillaumerose) [reply cmd/crc/cmd/oc_env.go:35]: Yes this is a mistake, thanks! 
Should I drop this? Hardcode the domain? It is sad to have crc oc-env failing this often only for proxy setup (which are rare setup). 
Should this check only performed when a proxy is enabled ?

**Comment by gbraad** [comment]
Oc-env is only intended for use with the CRC environment, which will only be useful when a VM with deployed bundle exists.

So, what is the idea here?

**Review by gbraad — CHANGES_REQUESTED** [review]
See previous comments as I ma not sure about the intentions here.

File: cmd/crc/cmd/oc_env.go (hunk 1 above)
Comment (guillaumerose) [reply cmd/crc/cmd/oc_env.go]: I dropped the commit. Too complicated for little result.

**Comment by guillaumerose** [comment]
The idea here was to have `crc oc-env` always working. Or at least in 90% of cases.

**Comment by guillaumerose** [comment]
I only kept the small cleanup commit.

File: cmd/crc/cmd/oc_env.go (hunk 1 above)
Comment (cfergeau) [reply cmd/crc/cmd/oc_env.go]: > Hardcode the domain? It is sad to have crc oc-env failing this often only for proxy setup (which are rare setup).
Should this check only performed when a proxy is enabled ?

A mix of these suggestions could work? If proxy is configured, try to get proxy info through machine metadata, if this fails, fallback to hardcoded domain, and warn that NO_PROXY could be incorrect?

**Review by gbraad — APPROVED** [review]

File: cmd/crc/cmd/oc_env.go (hunk 1 above)
Comment (guillaumerose) [reply cmd/crc/cmd/oc_env.go]: Will do that in a follow up

**Pull request merged**